src/app/components/ConsentBanner/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
```

Comment:
Could this component go in psammead too? Not too bothered if it does now or we make an issue for it though. When WS make their amp pages theyre gunna need this component too

Reply:
Yes once we have this and all the analytics work done, we can move it over into Psammead.